dist/embed.js

```diff
@@ -0,0 +1,356 @@
+
+  embedDiv.innerHTML = content;
+}
+
+document
+  .getElementsByTagName("head")[0]
+  .insertAdjacentHTML(
+    "beforeend",
+    '<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />'
+  );
+
+let momentScriptReady = false;
+const momentScript = document.createElement("script");
+momentScript.type = "text/javascript";
+momentScript.onload = () => {
+  momentScriptReady = true;
+  if (momentScriptReady && countdownScriptReady) start();
+};
+momentScript.src =
+  "https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.24.0/moment.min.js";
+document.head.appendChild(momentScript);
+
+let countdownScriptReady = false;
+const countdownScript = document.createElement("script");
+countdownScript.type = "text/javascript";
+countdownScript.onload = () => {
+  countdownScriptReady = true;
+  if (momentScriptReady && countdownScriptReady) start();
+};
+countdownScript.src =
+  "https://cdnjs.cloudflare.com/ajax/libs/countdown/2.6.0/countdown.min.js";
+document.head.appendChild(countdownScript);
+
+function start() {
+  createEmbedElements();
+
+  function selectElement(id, valueToSelect) {
```

Comment:
Looks like from this point on the code is almost the same as scripts/index.js is there a way that we can have index.js reference code in here? Best to keep things as DRY as possible